src/binary.ts

```diff
@@ -255,6 +255,35 @@ export class Binary {
     }
     return new Binary(data, type);
   }
+
+  /** @internal */
+  [Symbol.for('nodejs.util.inspect.custom')](): string {
+    return this.inspect();
+  }
+
+  inspect(): string {
+    const asBuffer = this.value(true);
+    switch (this.sub_type) {
+      case Binary.SUBTYPE_MD5:
+        return `MD5("${asBuffer.toString('hex')}")`;
+      case Binary.SUBTYPE_UUID:
+        if (asBuffer.length === 16) {
+          // Format '0123456789abcdef0123456789abcdef' into
+          // '01234567-89ab-cdef-0123-456789abcdef'.
+          const hex = asBuffer.toString('hex');
+          const asUUID = hex
+            .match(/^(.{8})(.{4})(.{4})(.{4})(.{12})$/)
+            ?.slice(1, 6)
+            .join('-');
+          return `UUID("${asUUID}")`;
+        }
+      // Fall through in case somebody did something weird and used an UUID
+      // with a non-standard length.
+      // eslint-disable-next-line no-fallthrough
+      default:
+        return `BinData(${this.sub_type}, "${asBuffer.toString('base64')}")`;
```

Comment:
@rose-m These are also shell types, so I think at least this one would probably be best changed as well (and, because the different names aren’t enough confusing, the arguments work pretty differently … `BinData()` in the shell takes subtype + base64 string in that order, `Binary` in BSON takes a “““binary””” string or buffer + subtype, i.e. in reverse order)… I think ideally this line could be something like 
```suggestion
        return `Binary(Buffer.from("${asBuffer.toString('base64')}", "base64"), ${this.sub_type})`;
```

but it sure isn’t pretty…